source/puppet/4.2/reference/release_notes.markdown

```diff
@@ -20,19 +17,91 @@ Make sure you also read the [Puppet 4.0 release notes](/puppet/4.0/reference/rel
 
 Also of interest: the [Puppet 4.1 release notes](/puppet/4.1/reference/release_notes.html).
 
+## Puppet 4.2.3
+
+Released October 29, 2015
+
+Shipped in puppet-agent version 1.2.7.
+
+Puppet 4.2.3 is a bug fix and platform support release in the Puppet 4.2 series. It also removes support for Ubuntu 14.10 (Utopic).
+
+For JIRA issues related to Puppet 4.2.3, see:
+
+* [Fixes for Puppet 4.2.3]()
+* [Introduced in Puppet 4.2.3]()
```

Comment:
We were talking about creating one query for all issues fixed and introduced in puppet-agent, which would include all of the components. We can create individual queries for each component, though it is going to be error-prone once we add more components, e.g. pxp-agent. We could automate that, but we're not there yet. Thoughts?